src/Api/AdminConsole/Controllers/GroupsController.cs

```diff
@@ -16,7 +17,7 @@
 
 namespace Bit.Api.AdminConsole.Controllers;
 
-[Route("organizations/{orgId}/groups")]
+[Route("organizations/{orgId}")]
```

Comment:
The top level was removed since the new route would be `organizations/{id}/group-details` since group details isn't following `/groups`. I believe the only way to override that would be to put a `/` at the start of the route, but then you'd override any additional route pathing that was set up higher in the stack.

Reply:
In that case I suggest leaving `groups` in the top-level route and using `details` at the method level, i.e. `organizations/{id}/groups/details`. (We already do this for collection details.) My main concern is that all our controllers use a controller-level route, this would be an outlier that is easily missed. This also keeps all groups-related resources under the `groups` path which I think might be better REST design (although that might be arguable).

Reply:
I see I specified `group-details` in the Jira ticket, that's my bad!

Reply:
No worries. Easy fix. I'll move group back to the top and make this one details. I'll update the client code accordingly as well.